app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -535,28 +536,41 @@ function handleImageUpload(event, randomIdNumber) {
   }
 }
 
+function toggleItemSummaryHtml(item) {
+  var itemSummaryContent = item.getElementsByClassName("js-collapse-comment-content")[0];
+  var usernames = item.getElementsByClassName("js-comment-username");
+  var number = "";
+  if (usernames.length > 1) {
+    number = " + " + (usernames.length - 1) + " replies"
+  }
+  var itemUsername = usernames[0].textContent + number
+  if (item.open) {
+    itemSummaryContent.innerHTML = "";
+  } else {
+    itemSummaryContent.innerHTML = itemUsername;
+  }
+  item.getElementsByTagName("SUMMARY")[0].blur();
```

Comment:
refactor (blocking) - This wasn't introduced in this PR, but I think we should remove this line. Blurring the summary after clicking it causes poor user experience for anyone who navigates by keyboard or uses a screen reader (as focus is sent back to the top of the page)